### PR TITLE
Hotfix 5338

### PR DESCRIPTION
**Pull request by jlhicks4925**
### Redmine ticket(s)
https://devops.dchbx.org/redmine/issues/5338

### Local build result
bundle exec rake parallel:spec[4]
Finished in 9 minutes 58 seconds (files took 9.53 seconds to load)
3998 examples, 0 failures, 78 pending

bundle exec cucumber
54 scenarios (54 passed)
964 steps (964 passed)
6m51.266s

### Latest rebase/merge tag
tag: 1.8.31

---

### Data ticket(s) Followup
* (redmine links here - optional)

### Related Pull Requests
* (github links here - optional)

---

#### Peer Review
* (For code review)

#### Functional Testing
* (For testing locally)

#### Deployment
* (for release manager and/or build manager)

### CODE REVIEW

File: app/models/special_enrollment_period.rb [hunk 1]
```diff
@@ -30,15 +33,39 @@ class SpecialEnrollmentPeriod
...
+  field :option3_date, type: Date
+
+  # CSL#
+  field :csl_num, type: String
+
+  # MARKET KIND
+  field :market_kind, type:String
+
+  validates :csl_num,
+    length: { minimum: 5, maximum: 10, message: "should be a minimum of 5 digits" },
```
Comment (Kevin-Wei) [inline app/models/special_enrollment_period.rb]: @jlhicks4925 @withintheruins14 is there any reason this validation message doesn't alert out both the minimum and maximum?

File: app/views/exchanges/hbx_profiles/sep_events.js.erb [hunk 2]
```diff
@@ -0,0 +1 @@
+alert("Hello! I am an alert box!!");
```
Comment (Kevin-Wei) [inline app/views/exchanges/hbx_profiles/sep_events.js.erb]: @withintheruins14 ??? @jlhicks4925 ???

File: app/views/exchanges/hbx_profiles/shared/_primary_nav.html.erb [hunk 3]
```diff
@@ -75,23 +75,32 @@
           <% end %>
         </li>
```
Comment (Kevin-Wei) [inline app/views/exchanges/hbx_profiles/shared/_primary_nav.html.erb]: Why was this spacing changed at all? @jlhicks4925

**Comment by davidplappert** [comment]
@withintheruins14 is this ticket still valid? This Redmine ticket links to another PR (#331)

**Comment by Kevin-Wei** [comment]
addressed via #671

**Pull request closed without merging**